source/running-mongosync/migration.txt

```diff
@@ -0,0 +1,10 @@
+.. _c2c-run-migration:
+
+=========
+Migration
```

Comment:
This is migration as opposed to synchronization - a one time thing, instead of an on-going thing. 

I think the common case will be onprem-atlas, but migration could be any of the uses, onprem-onprem, onprem-atlas, atlas-atlas.

Reply:
^ correct. It's migration to the same (e.g. onprem->onprem, or Atlas->Atlas) or hybrid (eg. onprem->Atlas, Atlas->onprem) environments.

Reply:
@alan-zheng-mdb @davemungo What about "Migrate Data to Another Cluster"?

Reply:
Sure, that works

Reply:
@alan-zheng-mdb does that work for you? I think this is an opportunity to make this a little more obvious